CONTRIBUTING.md

```diff
@@ -127,6 +155,37 @@ Standard prefixes for ITK commit messages:
   * `STYLE:` No logic impact (indentation, comments)
   * `WIP:` Work In Progress not ready for merge
 
+The body of the message should clearly describe the motivation of the commit
+(**what**, **why**, and **how**). In order to ease the task of reviewing
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to generate
```

Comment:
*to generate smooth release notes* -> *smooth generation of release notes*

Reply:
Done in 56ed51a.